client/src/app/+admin/moderation/video-abuse-list/video-abuse-details.component.ts

```diff
@@ -11,6 +13,39 @@ import { ProcessedVideoAbuse } from './video-abuse-list.component'
 export class VideoAbuseDetailsComponent {
   @Input() videoAbuse: ProcessedVideoAbuse
 
+  private predefinedReasonsTranslations: { [key: number]: string }
+
+  constructor (
+    private i18n: I18n
+  ) {
+    this.predefinedReasonsTranslations = {
+      [VideoAbusePredefinedReasons.VIOLENT_OR_REPULSIVE]: this.i18n('Violent or Repulsive'),
+      [VideoAbusePredefinedReasons.HATEFUL_OR_ABUSIVE]: this.i18n('Hateful or Abusive'),
+      [VideoAbusePredefinedReasons.SPAM_OR_MISLEADING]: this.i18n('Spam or Misleading'),
+      [VideoAbusePredefinedReasons.PRIVACY]: this.i18n('Privacy'),
+      [VideoAbusePredefinedReasons.RIGHTS]: this.i18n('Rights'),
+      [VideoAbusePredefinedReasons.SERVER_RULES]: this.i18n('Server rules'),
+      [VideoAbusePredefinedReasons.THUMBNAILS]: this.i18n('Thumbnails'),
+      [VideoAbusePredefinedReasons.CAPTIONS]: this.i18n('Captions')
+    }
+  }
+
+  get predefinedReasons () {
```

Comment:
I'm not sure to understand why you always use getters. I don't really like that because it's confusing and we don't really know in the template if this is a variable (with no cost) or a function (that could have an important complexity).

I'm not against getter when it's an alias to another variable or a small check. But it's not the case here.